src/integrationTest/java/uk/gov/hmcts/reform/idam/bdd/RetrySteps.java

```diff
@@ -39,8 +38,12 @@ public void itShouldRetryMakingIdamCall() {
         wiremock.verify(WireMock.postRequestedFor(WireMock.urlPathEqualTo("/lease")));
     }
 
-    @Then("it should rethrow exception")
-    public void itShouldRethrowException() {
-        assertThrows(FeignException.InternalServerError.class, service::run);
+    @Then("it should catch exception")
+    public void itShouldCatchException() {
+        try {
+            service.run();
+        } catch (Exception e) {
+            assertNotNull(e.getMessage());
+        }
```

Comment:
I know we set it up ourselves, but what would happened if `service.run()` didn't throw an exception? e.g. we changed the code, but forgot to check/update the tests?